frontend/src/features/properties/map/research/update/UpdateResearchHeader.tsx

```diff
@@ -0,0 +1,67 @@
+import { HeaderField } from 'features/mapSideBar/tabs/HeaderField';
+import { Api_ResearchFile } from 'models/api/ResearchFile';
+import * as React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { prettyFormatDate } from 'utils';
+
+export interface IUpdateResearchHeaderProps {
+  researchFile?: Api_ResearchFile;
+}
+
+const UpdateResearchHeader: React.FunctionComponent<IUpdateResearchHeaderProps> = props => {
+  const lefttColumnWidth = '6';
+  const leftColumnLabel = '4';
```

Comment:
nit leftColumnLabelWidth

Reply:
personally, I think I'd rather these variables are removed and the values just inlined, otherwise I can't see the value and the usage in the same area.

Reply:
This just makes sure that all columns are the same instead of having to manually update each attribute